src/main/resources/swagger/rawls.yaml

```diff
@@ -11,10 +11,8 @@ info:
     url: 'http://opensource.org/licenses/BSD-3-Clause'
   termsOfService: 'http://www.github.com/broadinstitute/rawls'
 
-basePath: /api
```

Comment:
no longer have a standard base path for everything... there is one under /register